easybuild/easyconfigs/m/MariaDB/MariaDB-10.0.21-foss-2015b.eb

```diff
@@ -1,27 +1,24 @@
 easyblock = 'CMakeMake'
 
 name = 'MariaDB'
-version = '5.5.29'
```

Comment:
no, you're changing an easyconfig that's now in develop (and which works fine): https://github.com/hpcugent/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/m/MariaDB